### ISSUE TITLE
Can't install io package

### ISSUE THREAD

**Issue by ssiccha** [oscar-system/GAP.jl#587]
I'm using the GAP start-up script pointed to by `GAP.gap_exe()` and want to load the io package. Installing it via the PackageManager yields
```
gap> InstallPackage("io");
#I  Getting PackageInfo URLs...
#I  Created directory /home/sergio/.gap/pkg/
#I  Retrieving PackageInfo.g from https://gap-packages.github.io/io/PackageInfo.g ...
#I  Downloading archive from URL https://github.com/gap-packages/io/releases/download/v4.7.0/io-4.7.0.tar.gz ...
#I  Saved archive to /tmp/tmk9768L/io-4.7.0.tar.gz
#I  Extracting to /home/sergio/.gap/pkg/io-4.7.0 ...
#I  Running compilation script on /home/sergio/.gap/pkg/io-4.7.0 ...
#I  Package availability test failed
#I  (for IO 4.7.0)
#I  Removed directory /home/sergio/.gap/pkg/io-4.7.0
false
```

**Comment by fingolfin** [comment]
I recommend to install packages via the Julia command `GAP.Packages.install("io")`.

I am not sure why PackageManager fails like that, but it overall is a bit fragile, which is why I am looking into replacing it with the Julia package manager.

**Comment by ssiccha** [comment]
Installing io via Julia also fails for me:
```
julia> GAP.Packages.install("io")
#I  Getting PackageInfo URLs...
#I  Retrieving PackageInfo.g from https://gap-packages.github.io/io/PackageInfo.g ...
#I  Downloading archive from URL https://github.com/gap-packages/io/releases/download/v4.7.0/io-4.7.0.tar.gz ...
#I  Saved archive to /tmp/tme3g2Lo/io-4.7.0.tar.gz
#I  Extracting to /home/sergio/.julia/gaproot/v4.11/pkg/io-4.7.0 ...
#I  Running compilation script on /home/sergio/.julia/gaproot/v4.11/pkg/io-4.7.0 ...
#I  Package availability test failed
#I  (for IO 4.7.0)
#I  Removed directory /home/sergio/.julia/gaproot/v4.11/pkg/io-4.7.0
false
```

**Comment by mohamed-barakat** [comment]
I encountered this problem some time ago but now it works fine:

https://github.com/homalg-project/CapAndHomalg.jl/runs/1542994393?check_suite_focus=true#step:4:186

You may want to have a look at this post and the discussion below:

https://github.com/oscar-system/GAP.jl/pull/550#pullrequestreview-542247974

**Comment by ssiccha** [comment]
Thanks for linking that @mohamed-barakat! My `~/.gap/gaprc` printed an error message which caused the error since I hadn't pulled @ThomasBreuer s fix  gap-system/gap#4198 yet. Now it works. :-)
